<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChipClass</class>
 <widget class="QMainWindow" name="ChipClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chip</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>文件名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fileName">
         <property name="text">
          <string>testcasewash.txt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="read">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>读取</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="reset">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>复位</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="start">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>全部播放</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pause">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lastStep">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>上一步</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextStep">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>下一步</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="wash">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>清洗</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>初始化</string>
    </property>
    <addaction name="actionInit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionInit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Init</string>
   </property>
   <property name="toolTip">
    <string>初始设置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Chip.qrc"/>
 </resources>
 <connections/>
</ui>
